kolena/_experimental/dataset/__init__.py

```diff
@@ -13,8 +13,59 @@
 # limitations under the License.
 from kolena._experimental.dataset._dataset import fetch_dataset
 from kolena._experimental.dataset._dataset import register_dataset
+from kolena.workflow.annotation import BitmapMask
```

Comment:
I'm more inclined to relocate these datatypes into a new module, and not tied to `dataset` (as learned from `workflow`)